### PR TITLE
Adding Transitions to Inline Editor

### LINKED ISSUE #875: Animate transitions for Inline Editor
Right now the lack of transitions makes edits feel too abrupt.

### PR DESCRIPTION
**Pull request by gvn**
Resolves #875 

R? @k88hudson

### CODE REVIEW

**Comment by thisandagain** [comment]
This is awesome. :smile: :metal:

File: components/blockList/index.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+            elSwapTarget.addEventListener(transitionEndEventName, function () {
+                transitionsCompleted++;
+                animationEnd();
+            });
+
+            // Move elements
+
+            if (steps === 1) {
+                elTarget.style.top = elSwapTargetHeight;
```
Comment (k88hudson) [inline components/blockList/index.js]: Would it be possible to use `translateY` instead of `top` here? translateX/Y are much more performant

**Comment by k88hudson** [comment]
looks AWESOME, small performance nit :P

**Comment by gvn** [comment]
Good call!

**Pull request merged**